test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml

```diff
@@ -0,0 +1,66 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.pipeline_alerts.yaml
+
+tests:
+  # ----- Tekton Controller Rapid Restart Tests ----
```

Comment:
still need to create the deadlock tests @amisstea but I was going to pattern those off the restart one, so let's see if you agree with what I did here and then I'll replicate

thanks